.github/workflows/test.yml

```diff
@@ -20,4 +20,4 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: ./
+    # - uses: ./
```

Comment:
We shouldn't do this. We need to validate our action works as a black box, not just via unit tests